mlflow/java/client/pom.xml

```diff
@@ -31,6 +31,16 @@
       <groupId>log4j</groupId>
       <artifactId>log4j</artifactId>
       <version>1.2.17</version>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.slf4j</groupId>
+      <artifactId>slf4j-log4j12</artifactId>
```

Comment:
Just to make sure I understand - after this change the MLflow client will by default continue to use log4j to log messages, but users can override this by adding a different slf4j binding to their classpath (?)